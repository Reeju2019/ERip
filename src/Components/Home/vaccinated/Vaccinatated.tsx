import React from "react";
import data from "../../../Data/const.data.json";
import "./vaccine.css";

const Vaccinatated = () => {
  return (
    <div>
      <section className="vaccinated">
        <div className="container">
          <div className="row">
            {data.vaccinated.map((c: any) => {
              return (
                <div className="caro-2 d-flex">
                  <div className="text-vaccinated justify-content-start col-8">
                    <h4 className="vaccinated-para">{c.para}</h4>
                    <h1 className="vaccinated-heading">
                      <strong>
                        {c.text1} <br />
                        {c.text2}
                      </strong>
                    </h1>
                  </div>
                  <img
                    src={c.img}
                    className="justify-content-end col-4 vaccinated-img"
                    alt=""
                  />
                </div>
              );
            })}
          </div>
        </div>
      </section>
    </div>
  );
};

export default Vaccinatated;
