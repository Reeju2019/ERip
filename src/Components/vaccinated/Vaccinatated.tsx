import React from "react";
import vaccineimg from "./img-fleet.png";
import "./vaccine.css";

const Vaccinatated = () => {
  return (
    <div>
      <section className="vaccinated">
        <div className="container">
          <div className="row">
            <div className="caro-2 d-flex">
              <div className="text-vaccinated justify-content-start col-8">
                <h4 className="vaccinated-para">#WeCare</h4>
                <h1 className="vaccinated-heading">
                  <strong>
                    Our Fleet is <br />
                    Fully Vaccinated!
                  </strong>
                </h1>
              </div>

              <img
                src={vaccineimg}
                className="justify-content-end col-4 vaccinated-img"
                alt=""
              />
            </div>
          </div>
        </div>
      </section>
    </div>
  );
};

export default Vaccinatated;
