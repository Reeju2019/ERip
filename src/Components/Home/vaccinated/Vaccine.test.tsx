import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import Vaccinatated from './Vaccinatated'

describe('pnf rendering specification', () => {
  test('vaccinated', () => {
    render(<Vaccinatated />)
  })
  it('SnapShot testing', () => {
    const component = renderer.create(<Vaccinatated />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<Vaccinatated />)
    ReactDOM.unmountComponentAtNode(div)
  })
})
