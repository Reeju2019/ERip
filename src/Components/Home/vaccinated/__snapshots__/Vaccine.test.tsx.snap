// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pnf rendering specification SnapShot testing 1`] = `
<div>
  <section
    className="vaccinated"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="caro-2 d-flex"
        >
          <div
            className="text-vaccinated justify-content-start col-8"
          >
            <h4
              className="vaccinated-para"
            >
              #WeCare
            </h4>
            <h1
              className="vaccinated-heading"
            >
              <strong>
                Our Fleet is
                 
                <br />
                Fully Vaccinated!
              </strong>
            </h1>
          </div>
          <img
            alt=""
            className="justify-content-end col-4 vaccinated-img"
            src="assets/img-fleet.png"
          />
        </div>
      </div>
    </div>
  </section>
</div>
`;
